fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -67,6 +67,8 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
         config.cli.local_mode = False
         for command in api_commands:
             cli.add_command(command)
+    else:
+        config.cli.local_mode = True
```

Comment:
Just a nitpick, but the `else` condition can be removed if this line is moved above the `if` statement on L66.

Reply:
however it was before was broken, I'll leave it as-is for now since its confirmed working